x-pack/test/apm_api_integration/tests/services/top_services.ts

```diff
@@ -91,37 +91,37 @@ export default function ApiTest({ getService }: FtrProviderContext) {
           Array [
             Object {},
             Object {
-              "latency": 496794.054441261,
-              "throughput": 11.6333333333333,
-              "transactionErrorRate": 0.0315186246418338,
+              "latency": 520294.126436782,
+              "throughput": 11.6,
+              "transactionErrorRate": 0.0316091954022989,
             },
             Object {
-              "latency": 83395.638576779,
-              "throughput": 17.8,
-              "transactionErrorRate": 0.00936329588014981,
+              "latency": 74805.1452830189,
+              "throughput": 17.6666666666667,
+              "transactionErrorRate": 0.00566037735849057,
             },
             Object {
-              "latency": 430318.696035242,
-              "throughput": 7.56666666666667,
-              "transactionErrorRate": 0.092511013215859,
+              "latency": 411589.785714286,
+              "throughput": 7.46666666666667,
+              "transactionErrorRate": 0.0848214285714286,
             },
             Object {
-              "latency": 53147.5747663551,
-              "throughput": 7.13333333333333,
+              "latency": 53906.6603773585,
+              "throughput": 7.06666666666667,
               "transactionErrorRate": 0,
             },
             Object {
-              "latency": 419826.24375,
+              "latency": 420634.9,
               "throughput": 5.33333333333333,
               "transactionErrorRate": 0.025,
             },
             Object {
-              "latency": 21520.4776632302,
-              "throughput": 9.7,
-              "transactionErrorRate": 0.00343642611683849,
+              "latency": 40989.5802047782,
```

Comment:
Latency went from `21520.4776632302` to `40989.5802047782`. Probably fine but would be good if we can double check.